apps/emqx_bridge_kafka/src/emqx_bridge_kafka.erl

```diff
@@ -294,7 +294,37 @@ fields(producer_kafka_opts) ->
             mk(ref(producer_buffer), #{
                 required => false,
                 desc => ?DESC(producer_buffer)
-            })}
+            })},
+        {query_mode_sync_timeout,
```

Comment:
rename to `sync_query_timeout`, and move it to the `resource_opts_fields` struct.

Reply:
I renamed the field but the `resource_opts_fields` struct does not exist after the refactoring I did based on your suggestion. Please let me know if you still think it needs to be changed.